lib/template-assembler.js

```diff
@@ -7,6 +7,29 @@ const upath = require('upath');
 const partialRegex = /\{\{>\s*([_|\-|a-zA-Z0-9/]+)[^{]*?}}/g;
 const dynamicComponentRegex = /\{\{\s*?dynamicComponent\s*(?:'|")([_|\-|a-zA-Z0-9/]+)(?:'|").*?}}/g;
 
+/**
+ * Takes a templates folder and template name. It returns simple path for custom template if it's available
+ *  or use default folder instead
+ *
+ * @param templatesFolder
+ * @param templateName
+ */
+function getCustomPath(templatesFolder, templateName) {
+    const packageFlag = 'external/';
```

Comment:
I believe it's not a flag. A `flag` is usually considerer a boolean variable.

Reply:
it's changed to `packageMarker` since it indicates which source has been used for template. 
@jairo-bc  how do you find this name?